src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/search-utils.ts

```diff
@@ -7,6 +7,9 @@ export enum ExtendedSearchBehavior {
     WildcardOnEveryTerm = 'WildcardOnEveryTerm',
     SimplifiedSearchOr = 'SimplifiedSearchOr',
     AdvancedSearch =  'AdvancedSearch',
+    CPSimplifiedSearch = 'simplified',
```

Comment:
In my opinion, it would make sense to have these 3 options in a separate CP enum.
And then we could also have a map function in this file instead of the switch case in the search-box.component.ts -> ngOnChanges

Reply:
The ExtendedSearchBehavior is also being used in the DataExchange  app, there are situations where a variable can have anyone of those values, so separating them would add some typing complexity to those variables